types/signing.go

```diff
@@ -24,7 +24,7 @@ type SigningData struct {
 	Domain Domain `ssz-size:"32"`
 }
 
-type forkData struct {
+type ForkData struct {
```

Comment:
Was this intentionally named `forkData` instead of `ForkData` (capitalized)? If so, I will revert my changes to it.

Reply:
I think it's allright to export it 👍